accounts/usbwallet/wallet.go

```diff
@@ -632,7 +633,7 @@ func (w *wallet) SignTx(account accounts.Account, tx *types.Transaction, chainID
 // data is not supported for Ledger wallets, so this method will always return
 // an error.
 func (w *wallet) SignTextWithPassphrase(account accounts.Account, passphrase string, text []byte) ([]byte, error) {
-	return w.SignText(account, text)
+	return w.SignText(account, accounts.TextHash(text))
```

Comment:
This change seems to conflict with what’s already in master.  SignTextWithPassphrase was updated in https://github.com/ethereum/go-ethereum/pull/33138. A rebase should fix it.